documentation/book/cluster-operator.adoc

```diff
@@ -515,6 +518,27 @@ Furthermore, containers doing lots of disk I/O (such as Kafka broker containers)
 for use as operating system page cache. On such containers, the request memory should be substantially more than the
 memory used by the JVM.
 
+[[kafka_rack]]
+===== Kafka rack
+
+It is possible to enable Kafka rack-awareness by specifying a JSON string for the `kafka-rack` field in the cluster ConfigMap.
+Such a JSON object has one mandatory field named `topologyKey`.
+This key needs to match one of the labels assigned to the {ProductPlatformName} cluster nodes.
+The label is used by {ProductPlatformName} when scheduling Kafka broker pods to nodes. 
+{ProductPlatformName} will try to minimize the number of pods scheduled on nodes with the same label value.
```

Comment:
This is not really true ... it has no impact on other pods. I think what {ProductPlatformName} does is nicely explained two lines below: _This will have the effect of spreading the broker pods across the availability zones_ We should use just that and throw away this sentence IMHO.